ds/list/list.go

```diff
@@ -173,11 +173,8 @@ func (l *List) Size(key string) (int, error) {
 // LRange returns the specified elements of the list stored at key
 // [start,end]
 func (l *List) LRange(key string, start, end int) (list [][]byte, err error) {
-	if l.IsExpire(key) {
-		return nil, ErrListNotFound
-	}
 	size, err := l.Size(key)
-	if err != nil {
+	if err != nil || size == 0 {
```

Comment:
Maybe warping this logic as a function is a better practice? Like l.empty()? WDYT?

Reply:
Sure, it will be more logical.

Reply:
@ShawnHXH Yep, I think your idea is great.

Reply:
Sounds great